metricbeat/metricbeat.yml

```diff
@@ -14,7 +14,7 @@ metricbeat.config.modules:
   path: ${path.config}/modules.d/*.yml
 
   # Set to true to enable config reloading
-  reload.enabled: false
+  reload.enabled: true
```

Comment:
This is a breaking change too, I'm wondering if we should recommend this mode by default or not. It may make things less predictable, wdyt?

Reply:
+1 for not enabling reload by default. If I were a user who is getting started and editing the configuration just to see what a Beat can do, I would not like it to restart without my explicit permission.

Reply:
@exekias Now that we have the making it GA out of the way, lets continue the discussion here. Any thoughts to the above?

Reply:
Good point about the breaking change, I guess it's not. I still don't like to have it enabled by default, I don't know many (or any) projects doing live changes without the user requesting them, it's probably unexpected.

Reply:
Ok, will close the PR then :-D